kv/store_vars.go

```diff
@@ -72,3 +74,17 @@ const (
 func (r ReplicaReadType) IsFollowerRead() bool {
 	return r != ReplicaReadLeader
 }
+
+// String implements fmt.Stringer interface.
+func (r ReplicaReadType) String() string {
+	switch r {
+	case ReplicaReadLeader:
+		return "leader"
+	case ReplicaReadFollower:
+		return "follower"
+	case ReplicaReadMixed:
+		return "mixed"
+	default:
+		return fmt.Sprintf("unknown-%v", byte(r))
```

Comment:
`ReplicaReadLearner` and `ReplicaReadPreferLeader` are missing.

Reply:
done.